crt_portal/static/sass/custom/proform.scss

```diff
@@ -0,0 +1,7 @@
+.crt-portal_date_inputs_btn {
+    display: flex;
+
+    .autofill_today_btn {
+        margin-top: auto;
```

Comment:
`margin-top` is one way to do this, but personally I prefer using flex. If you do, then this CSS declaration can also be removed: adding the class names `display-flex flex-align-end` will achieve the same effect (see [USWDS reference](https://designsystem.digital.gov/utilities/flex/) for details)